source/_integrations/tuya.markdown

```diff
@@ -36,38 +50,51 @@ The Tuya integration allows integrating all Powered by Tuya devices you have add
 - **Scene**: Smart scene.
 - **Switch**: Switch, socket, power strip, smart kettle, breaker, pet water feeder, air purifier, diffuser.
 
-## Quick installation and startup
-
-### Configure on the Tuya IoT Platform
+## Configuration of the Tuya IoT Platform
 
 #### Prerequisites
 
 You have created an account for [Tuya Smart or Smart Life app](https://developer.tuya.com/en/docs/iot/tuya-smart-app-smart-life-app-advantages?id=K989rqa49rluq#title-1-Download).
 
+You have created an account for the [Tuya IoT Platform](https://iot.tuya.com/).
+
 #### Create a project
 
+<p class='note'>
+If you haven't already signed up in Tuya IoT Platform, you will need to make an account first. You cannnot log in with your app's credentials here.
+</p>
+
 **1.** Log in to the [Tuya IoT Platform](https://iot.tuya.com/).
 
-**2.** In the left navigation bar, click **Cloud** > **Projects**.
+**2.** In the left navigation bar, click **Cloud** > **Development**.
 
-**3.** On the page that appears, click **Create**.
+<p class='note'>
```

Comment:
Does this have to be a note block? Or can we just make this part of the text?

Reply:
Nope it does not